### DISCUSSION TITLE
How many hours will be put into servo per week?

### DISCUSSION THREAD

**Discussion by KnorpelSenf** [servo/servo#30221] [General] [answered]
In your blog post from January (https://servo.org/blog/2023/01/16/servo-2023/), it was said:

> This year, thanks to new external funding, a team of developers will be actively working on Servo.

Can you share how many team members there are/will be, and how much time they can spend on this project?

I'm very excited to see this thing pick up more speed in the future!

**Comment by mukilan** [comment, marked as answer]
Hi @KnorpelSenf,

There are 4-6 developers working full-time on Servo currently. This number might change in the future based on how the project evolves.

You may want to checkout our most recent [presentation](https://www.youtube.com/watch?v=pfk8s5OD99A) to learn more about the roadmap and goals for Servo.

**Reply by KnorpelSenf** [reply]
Already watched it, that's what made me curious! Thanks for the response :)

**Reply by Abdillah** [reply]
> There are 4-6 developers working full-time on Servo currently.

Wow, full time? That's great.

I'm interested in making Servo into Electron, then create a configurable electron browser like [Vieb](https://github.com/Jelmerro/Vieb). But Servo live-ness is something I hoping for.
